xml/System.Net.Sockets/ProtocolFamily.xml

```diff
@@ -919,11 +893,11 @@
     </Member>
     <Member MemberName="Packet">
       <MemberSignature Language="C#" Value="Packet" />
-      <MemberSignature Language="ILAsm" Value=".field public static literal valuetype System.Net.Sockets.ProtocolFamily Packet = int32(31)" />
+      <MemberSignature Language="ILAsm" Value=".field public static literal valuetype System.Net.Sockets.ProtocolFamily Packet = int32(65536)" />
       <MemberSignature Language="DocId" Value="F:System.Net.Sockets.ProtocolFamily.Packet" />
       <MemberSignature Language="VB.NET" Value="Packet" />
       <MemberSignature Language="C++ CLI" Value="Packet" />
-      <MemberSignature Language="F#" Value="Packet = 31" Usage="System.Net.Sockets.ProtocolFamily.Packet" />
+      <MemberSignature Language="F#" Value="Packet = 65536" Usage="System.Net.Sockets.ProtocolFamily.Packet" />
```

Comment:
This is a suspicious change, but I verified it in the source. That's true of the other changes to underlying enum values as well -- all are enum members introduced in 3.0.